tests/acceptance/features/webUIManageUsersGroups/editUsers.feature

```diff
@@ -0,0 +1,35 @@
+@webUI @insulated @disablePreviews
+Feature: edit users
+  As an admin
+  I want to edit users
+  So that I can manage users properly
+
+  Background:
+    Given user admin has logged in using the webUI
+
+  Scenario: Admin changes the display name of the user
+    Given user "user0" has been created with default attributes
+    And the administrator has browsed to the users page
+    When the administrator changes the display name of user "user0" to "New User" using the webUI
+    And the administrator logs out of the webUI
+    And user "user0" logs in using the webUI
+    Then "New User" should be shown as the name of the current user on the WebUI
+    And user "user0" should exist
+    And the user attributes returned by the API should include
+    | displayname | New User |
+
+  Scenario: Admin changes the password name of the user
+    Given user "user0" has been created with default attributes
+    And the administrator has browsed to the users page
+    When the administrator changes the password of user "user0" to "new_password" using the webUI
+    Then user "user0" should exist
+    And the content of file "textfile0.txt" for user "user0" using password "new_password" should be "ownCloud test text file 0" plus end-of-line
+    But user "user1" using password "%regular%" should not be able to download file "textfile0.txt"
+
+  Scenario: Admin adds a user to a group
```

Comment:
Also make a scenario that has multiple groups created, and the user is added to one of them. That will test that correct group can be selected from the UI list. (The current scenario is good for testing the case when there is only 1 group that exists)